Mathlib/Topology/ContinuousFunction/UniqueCFC.lean

```diff
@@ -96,6 +96,7 @@ variable {A : Type*} [TopologicalSpace A] [Ring A] [StarRing A] [Algebra ℝ A]
 
 namespace StarAlgHom
 
+set_option synthInstance.maxHeartbeats 21000 in
```

Comment:
Is this still necessary?

Reply:
I tried removing it, let's see if it works